android/titanium/src/java/org/appcelerator/titanium/proxy/ActivityProxy.java

```diff
@@ -25,6 +29,8 @@
 import android.content.Intent;
 import android.os.Message;
 import android.support.v7.app.AppCompatActivity;
```

Comment:
Can we clean up a few unused imports? I would like to note that they did not become such after these changes, but it may be a good opportunity to optimize them.